survey_link_base/models/survey_link_mixin.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Binhex - Adasat Torres de León
+# Copyright 2024 Binhex - Zuzanna Elzbieta Szalaty Szalaty
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from datetime import timedelta
+
+from odoo import _, api, fields, models
+
+
+class SurveyLinkMixin(models.AbstractModel):
+    _name = "survey.link.mixin"
+
+    survey_count = fields.Integer(
```

Comment:
survey_count here is ambiguous because it implies survey but computes survey answers, maybe rename field and compute method name?